src/perf/lib/PerfServer.cpp

```diff
@@ -71,6 +73,24 @@ PerfServer::Init(
         }
     }
 
+    _Benign_race_begin_
+    if (TryGetValue(argc, argv, "delay", &this->DelayMicroseconds)) {
```

Comment:
FYI, we have `TryGetVariableUnitValue` in `perfclient.cpp` that allows for the app to specify its own time unit, such as `-delayType:2ms` or `-delayType:100us`. I think it would be useful here.